CONTRIBUTING.md

```diff
@@ -1,5 +1,34 @@
 # Contributing to Open API Renderer
 
+Thanks being interested in contributing to this project! It's Temando's first open source project so bare with us while we iron out the kinks. The following describes some basic information that should help you get started.
+
+## Browser/device support
+
+This project aims to support evergreen browsers, such as Chrome and Firefox. Support for IE and Safari versions is to be determined, but likely will start IE 11/Safari 9.
```

Comment:
You might consider Safari evergreen. They release updates independent of the OS: https://support.apple.com/en-au/HT201222 (on the desktop, anyway).